tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -707,6 +718,46 @@ private static boolean shouldIgnoreUri(String uri) {
     return ImmutableMultimap.copyOf(builder);
   }
 
+  // Combines all script tags into one and append it at the bottom of the
+  // document.
+  private static void combineScriptTags(Document document) {
```

Comment:
nit: These are script elements, not script tags (the tag is just the
literal `<script>` part).